src/app/test_executive/verification_key_update.ml

```diff
@@ -344,7 +344,7 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
          the old key v1"
         (send_invalid_zkapp ~logger
            (Network.Node.get_ingress_uri whale1)
-           zkapp_command_update_vk2_refers_vk1 "Verification_failed" )
+           zkapp_command_update_vk2_refers_vk1 "invalid" )
```

Comment:
Can you add specific errors instead. This one most probably is `Unexpected_verification_key`. Ok for it to be in a follow-up PR

Reply:
I wasn't considering this, because `Verification_failed` wasn't specific at all. I'd create an issue for it if it's important.

Reply:
I think I fixed this, waiting for CI to confirm